bindings/c/test/fdb_api.hpp

```diff
@@ -640,6 +746,29 @@ class Transaction {
 		return native::fdb_transaction_watch(tr.get(), key.data(), intSize(key));
 	}
 
+	TypedFuture<future_var::GranuleDescriptionRefArray> readBlobGranulesDescription(KeyRef begin,
+	                                                                                KeyRef end,
+	                                                                                int64_t beginVersion,
+	                                                                                int64_t readVersion,
+	                                                                                int64_t* readVersionOut) {
+		return native::fdb_transaction_read_blob_granules_description(tr.get(),
+		                                                              begin.data(),
+		                                                              intSize(begin),
+		                                                              end.data(),
+		                                                              intSize(end),
+		                                                              beginVersion,
+		                                                              readVersion,
+		                                                              readVersionOut);
+	}
+
+	Result parseSnapshotFile(BytesRef fileData) {
```

Comment:
If these do not use the future we can declare them as static or maybe even outside Future.

Reply:
They do not use Transaction, but use a private constructor of Result which is Transaction's friend.